src/Microsoft.Health.Fhir.SqlServer/IndentedStringBuilder.cs

```diff
@@ -36,6 +37,60 @@ internal int IndentLevel
         /// <returns>A scope to be disposed when the indentation level is to be restored</returns>
         internal IndentedScope Indent() => new IndentedScope(this);
 
+        /// <summary>
+        /// Appends a column name to this instance.
+        /// </summary>
+        /// <param name="column">The column</param>
+        /// <returns>This instance</returns>
+        [Obsolete("Use overload with table alias instead")] // Catch calls an raise compiler warnings.
```

Comment:
Why not just remove this method?

Reply:
The idea was to catch cases in the future where we forget to call the overload with the alias. If we remove this, calls to `Append(myColumn)` will resolve the the object `Append(Object)` overload. We don't need to keep this, I was thinking it could be helpful.

Reply:
Got it, that makes sense.